deps.ffmpeg/99-ffmpeg.zsh

```diff
@@ -0,0 +1,294 @@
+      )
+      ff_cxxflags=(
+        -I"${target_config[output_dir]}"/include
+        -target "${arch}-apple-macos${MACOSX_DEPLOYMENT_TARGET}"
+        ${cxx_flags}
+      )
+      ff_ldflags=(
+        -L"${target_config[output_dir]}"/lib
+        -target "${arch}-apple-macos${MACOSX_DEPLOYMENT_TARGET}"
+        ${ld_flags}
+      )
+
+      args+=(
+        --cc=clang
+        --cxx=clang++
+        --host-cc=clang
+        --extra-libs="-lstdc++"
+        --arch="${arch}"
+        --enable-libaom
+        --enable-videotoolbox
+        --enable-pthreads
+        --enable-rpath
+      )
+
+      if [[ ${CPUTYPE} != "${arch}" ]] args+=(--enable-cross-compile)
+    ;;
+    linux-*)
+      ff_cflags=(
+        -I"${target_config[output_dir]}"/include
+        ${c_flags}
+      )
+      ff_cxxflags=(
+        -I"${target_config[output_dir]}"/include
+        ${cxx_flags}
+      )
+      ff_ldflags=(
+        -L"${target_config[output_dir]}"/lib
+        ${ld_flags}
+      )
+
+      args+=(
+        --arch="${arch}"
+        --enable-libaom
+        --enable-libsvtav1
+        --enable-pthreads
+        --extra-libs="-lpthread -lm"
+      )
+
+      if (( ${+commands[clang]} )) {
+        args+=(
+          --cc=clang
+          --cxx=clang++
+          --host-cc=clang
+        )
+      }
+
+      if [[ ${CPUTYPE} != "${arch}" ]] args+=(--enable-cross-compile)
+      ;;
+    windows-x*)
+      ff_cflags=(
+        -I"${target_config[output_dir]}"/include
+        -static-libgcc
+        ${c_flags}
+      )
+      ff_cxxflags=(
+        -I"${target_config[output_dir]}"/include
+        -static-libgcc
+        -static-libstdc++
+        ${cxx_flags}
+      )
+      ff_ldflags=(
+        -L"${target_config[output_dir]}"/lib
+        -static-libgcc
+        -static-libstdc++
+        ${ld_flags}
+      )
+
+      if (( ! shared_libs )) {
+        ff_ldflags+=(-Wl,-Bstatic -pthread)
+        autoload -Uz hide_dlls && hide_dlls
+      }
+
+      args+=(
+        --arch="${target_config[cmake_arch]}"
+        --target-os=mingw32
+        --cross-prefix="${target_config[cross_prefix]}-w64-mingw32-"
+        --pkg-config=pkg-config
+        --enable-cross-compile
+        --enable-w32threads
+        --disable-pthreads
+      )
+
+      if [[ ${arch} == 'x86_64' ]] args+=(--enable-libaom --enable-libsvtav1)
+    ;;
+  }
+
+  args+=(
+    --prefix="${target_config[output_dir]}"
+    --host-cflags="-I${target_config[output_dir]}/include"
+    --host-ldflags="-I${target_config[output_dir]}/include"
+    --extra-cflags="${ff_cflags}"
+    --extra-cxxflags="${ff_cxxflags}"
+    --extra-ldflags="${ff_ldflags}"
+    --enable-version3
+    --enable-gpl
+    --enable-libx264
+    --enable-libopus
+    --enable-libvorbis
+    --enable-libvpx
+    --enable-libtheora
+    --enable-libmp3lame
+    --enable-librist
+    --enable-libsrt
+    --enable-shared
+    --disable-static
+    --disable-libjack
+    --disable-indev=jack
+    --disable-outdev=sdl
```

Comment:
SDL maybe, but this is more important for POSIX systems where both dependencies might otherwise exist on the system and will be picked up by FFmpeg. Doesn't hurt to have those by default.